packages/config/constants.json

```
@@ -659,7 +659,7 @@
     "pol": {
       "bgtIncentiveDistributor": {
         "name": "BGTIncentiveDistributor",
-        "bepolia-address": "",
+        "bepolia-address": "0x77DA09bC82652f9A14d1b170a001e759640298e6",
```

```suggestion
        "bepolia-address": "0xb0d005Fe83E3F1ec876C1a64700c5F0d6265d9E3",
```
"mainnet-address": "0x77DA09bC82652f9A14d1b170a001e759640298e6",